.pre-commit-config.yaml

```diff
@@ -3,3 +3,4 @@ repos:
     rev: 1.17.0
     hooks:
       - id: prettier
+        args: ['--prose-wrap=always']
```

Comment:
_minor_: do we need it here if it's already specified in the config file?

Reply:
@shcheklein I tested this a little bit. I figured out that the pre-commit hook wasn't wrapping the text which could be because it was following the default config which is `proseWrap: preserve`. So I needed to explicitly pass the argument.
So yes, pre-commit hook doesn't read from `.prettierrc` file.